Workbooks/WorkbooksMetadata.json

```diff
@@ -493,4 +493,17 @@
     "subtitle": "",
     "provider": "Microsoft"
     }  
+      {
+    "workbookKey": "SecurityStatusWorkbook",
+    "logoFileName": "",
+    "description": "Security Staus dashboard for info About Azure and Azure Arc.",
+    "dataTypesDependencies": [ "Azure Resource Graph" ],

```

Comment:
This should be the names of the data types that are being queried.
For example: syslog,  CommonSecurityLog, SecurityEvent, etc.